enhancements/update/channel-rename-generally-available.md

```diff
@@ -0,0 +1,586 @@
+
+### Non-Goals
+
+* Achieve a completely painless transitional period.
+  While pain-reduction is good, an implementation where limited pain is expected is still acceptable.
+* Adjust `candidate-4.y` and `eus-4.y` names or semantics.
+  These channels are not relevant to this enhancement.
+
+## Proposal
+
+Beginning with 4.15, the `ga-4.y` and `fleet-approved-4.y` patterns will be initiated, making `ga-4.15` and `fleet-approved-4.15` the first in that line.
+`ga-4.y` will have the same semantics as `fast-4.y`, and `fleet-approved-4.y` will have the same semantics as `stable-4.y`.
+The `fast-4.y` and `stable-4.y` pattern will remain as public aliases, at least during 4.15, with [4.16 and later handling still under discussion][OTA-1221].
+
+### Workflow Description
+
+#### Updates into 4.15
+
+1. The cluster is running a 4.14.z release in a 4.14-capped channel like [`stable-4.14`][stable-4.14].
+1. The update service and cluster version operator [populate `status.desired.clusters`](#historical-context) with the channels compatible with the current 4.15.z, including the `*-4.14`, `candidate-4.15`, and other `*-4.15`.
+   New with this enhancement, those will include `ga-4.15` and `fleet-approved-4.15`.
+1. The cluster admin decides they want to update to 4.15.
+1. The cluster admin uses [4.15 `oc adm upgrade channel ...`][set-channel-4.15-oc] or [the 4.14 web-console][set-channel-4.14-web-console] to view the list of available channels.
+1. The cluster admin is surprised by the new `ga-4.15`, etc. and they click through to docs:
+  * [Web-consoles need an update][web-console-channel-doc-link].
+  * `oc adm upgrade [channel]` does not include a doc link at the moment.  [Should it?](#open-questions)  If so, what kind of version skew did we want this to work for?
+1. The cluster admin selects a 4.15-capped channel like `ga-4.15`.
+1. The cluster admin initiates an update to a 4.15.z release recommended by their new channel.
+
+#### 4.15 installs
+
+1. The cluster admin decides they want to install 4.15.
+1. The cluster admin acquires a 4.15.z installer binary.
+  * Signed installer binaries are available on the mirrors, e.g. [amd64-fast-4.14][mirror-fast-4.14].
+  * [Docs][install-docs] point at [console.redhat.com][console-create] which in turn [points][console-install-user-provisioned] into the mirrors with `stable` links like [this][mirror-stable-installer].
+    This path does not currently seem to point out the existence o [the sibling GPG signature][mirror-stable-signature].
+  * Installer binaries (with hashes) are also linked from [the OCP portal page][portal-ocp], with URIs that include the version and hashes, but which do not include any channel names.
+1. The cluster admin runs the installer, which generates a ClusterVersion manifest declaring [a default channel that is manually maintained][installer-default-channel].
+   For 4.15, the default will remain `stable-4.15`.
+   For 4.15, the default will pivot to `ga-4.16`.
```

Comment:
For 4.15 -> For 4.16 ?

Reply:
Thanks!  Pushed a fix with 569bf1ea -> c669a82a.